mention-bot.js

```diff
@@ -9,7 +9,17 @@
  * @flow
  */
 
-var downloadFileSync = require('download-file-sync');
+var githubWebClient = require('./githubWebClient');
+
+var downloadFileSync = function(url, cookies) {
+  var command = 'curl -L ' + url;
```

Comment:
can you also add --silent, I just pushed an update to download-file-sync to add this :)